src/scripts.js

```diff
@@ -524,3 +525,9 @@ function makeFriendChallengeHTML(id, activityInfo, userStorage, method) {
 function makeStepStreakHTML(id, activityInfo, userStorage, method) {
   return method.map(streakData => `<li class="historical-list-listItem">${streakData}!</li>`).join('');
 }
+
+function displaySuccessMessage(form) {
```

Comment:
Great idea to give the user some confirmation that their data was posted.  We may want to add some stylings to the message to make it more evident to the user though.  The text is currently small and is almost buried in the form at the moment.